lib/graphql/graphql-token-fee-fetcher.ts

```diff
@@ -36,10 +36,12 @@ export class GraphQLTokenFeeFetcher implements ITokenFeeFetcher {
     try {
       const tokenFeeResponse: TokensInfoResponse = await this.graphQLProvider.getTokensInfo(this.chainId, addresses)
       tokenFeeResponse.tokens.forEach((token) => {
-        if (token.feeData.buyFeeBps || token.feeData.sellFeeBps) {
+        if (token.feeData && (token.feeData.buyFeeBps || token.feeData.sellFeeBps)) {
```

Comment:
completely a nit, but this could also be: 
`if (token.feeData?.buyFeeBps || token.feeData?.sellFeeBps) {`